cub/test/catch2_test_launch_helper.h

```diff
@@ -59,11 +60,70 @@
 //!
 //! Consult with `test/catch2_test_cdp_wrapper.cu` for more usage examples.
 
-#if !defined(TEST_CDP)
-#error Test file should contain %PARAM% TEST_CDP cdp 0:1
+#if !defined(TEST_LAUNCH)
+#error Test file should contain %PARAM% TEST_LAUNCH lid 0:1
 #endif
 
-#if TEST_CDP == 1
+
+#if TEST_LAUNCH == 2
```

Comment:
proposal no change requested:

At some point in the near future we should move this from numerical constants to enumerations with a proper name